emission/net/api/cfc_webapp.py

```diff
@@ -263,6 +263,18 @@ def getSuggestion():
   logging.debug("Output of ret_dir = %s" % ret_dir)
   return ret_dir
 
+
+@post('/suggestion_sys')
+def getSingleTripSuggestion(tripid):
```

Comment:
proof that this is actually being invoked? You could add a log statement to this method with the trip id and put the log statements in here as part of the "Testing done" for example.

Reply:
Currently waiting on Sam to test the function, but will get logs in tonight or tomorrow morning.

Reply:
@valin1 you shouldn't have to wait on @sam11b to test your function. You should be able to test your own function using unittests or API tests.
e.g. once you have defined an API, you can call that API with `requests` just like you call the yelp API.

If you do that, then you know that your part works, and if there is an error, it must be in @sam11b's part

Reply:
@shankari  I wasn't sure how to run API tests? Because I tried to import the cfc_webapp as a module in the iPython notebook, but it wasn't working since it was unable to import that file. Are there any example API tests?

Reply:
@valin1 you have just defined an API like the yelp API. How do you make calls to the yelp API? Do you import a module from the yelp code?

Reply:
@valin1 did you figure out how to do this? I think it would really help you if you could test your code without waiting for others. And for long-term maintenance, it would be good to have API tests be part of this module.

Reply:
Invoked through demo and example logs from Sam, will work on making API tests

Reply:
can you copy-paste the logs that prove this?

Reply:
```
START 2019-01-25 18:55:10.366171 POST /suggestion_sys/getSing/5c3e58359258b9668aacbcc2
Entry({'_id': ObjectId('5c3e58359258b9668aacbcc2'), 'user_id': UUID('26596583-8320-43da-b27c-119775f2005b'), 'metadata': {'key': 'analysis/cleaned_trip', 'platform': 'server', 'write_ts': 1547589685.6595576, 'time_zone': 'America/Los_Angeles', 'write_local_dt': {'year': 2019, 'month': 1, 'day': 15, 'hour': 14, 'minute': 1, 'second': 25, 'weekday': 1, 'timezone': 'America/Los_Angeles'}, 'write_fmt_time': '2019-01-15T14:01:25.659558-08:00'}, 'data': {'source': 'DwellSegmentationDistFilter', 'end_ts': 1547587132.549234, 'end_local_dt': {'year': 2019, 'month': 1, 'day': 15, 'hour': 13, 'minute': 18, 'second': 52, 'weekday': 1, 'timezone': 'America/Los_Angeles'}, 'end_fmt_time': '2019-01-15T13:18:52.549234-08:00', 'end_loc': {'type': 'Point', 'coordinates': [-115.20650423647366, 36.279549129069075]}, 'raw_trip': ObjectId('5c3e58249258b9668aacbcbb'), 'start_ts': 1547585302.288933, 'start_local_dt': {'year': 2019, 'month': 1, 'day': 15, 'hour': 12, 'minute': 48, 'second': 22, 'weekday': 1, 'timezone': 'America/Los_Angeles'}, 'start_fmt_time': '2019-01-15T12:48:22.288933-08:00', 'start_loc': {'type': 'Point', 'coordinates': [-115.31409369973038, 36.11557410729392]}, 'duration': 1830.26030087471, 'distance': 32036.17486491815, 'start_place': ObjectId('5c3e4a299258b96620cfcc5f'), 'end_place': ObjectId('5c3e58369258b9668aacbd04')}})
{"coordinates": [-115.31409369973038, 36.11557410729392], "type": "Point"}
{"coordinates": [-115.20650423647366, 36.279549129069075], "type": "Point"}
END 2019-01-25 18:55:14.282497 POST /suggestion_sys/getSing/5c3e58359258b9668aacbcc2 26596583-8320-43da-b27c-119775f2005b 3.9162449836730957 
``` 
```
START 2019-01-25 18:55:46.814002 POST /suggestion_sys/getSug
Kroeber Hall, Bancroft Way, Southside, Berkeley, Alameda County, 94704, United States of America
Trilight Properties, College Avenue, Elmwood, Berkeley, Alameda County, 94704, United States of America
Oscar Wilde House, 2410, Warring Street, Panoramic Hill, Berkeley, Alameda County, 94704, United States of America
Trilight Properties, College Avenue, Elmwood, Berkeley, Alameda County, 94704, United States of America
Bancroft Way & Telegraph Av, Lower Sproul Plaza, Southside, Berkeley, Alameda County, 94704, United States of America
South Hall, South Hall Road, Northside, Berkeley, Alameda County, 94720, United States of America
Griffiths Hall, Dwight Way, Southside, Berkeley, Alameda County, 94704, United States of America
Martin Luther King Jr. Student Union, Lower Sproul Plaza, Southside, Berkeley, Alameda County, 94704, United States of America
Hildebrand Hall, Oppenheimer Way, Northside, Berkeley, Alameda County, 94720, United States of America
Memorial Pool, Sather Road, Northside, Berkeley, Alameda County, 94720, United States of America
Soda Hall, Le Roy Avenue, Northside, Berkeley, Alameda County, 94709, United States of America
College Av & Parker St, College Avenue, Elmwood, Berkeley, Alameda County, 94705, United States of America
END 2019-01-25 18:56:54.640763 POST /suggestion_sys/getSug 26596583-8320-43da-b27c-119775f2005b 67.82667660713196 
```